extension/show-names.js

```diff
@@ -12,7 +12,7 @@ window.showRealNames = () => {
 	const addUsersName = (user, name) => {
 		const $usernameLinks = $(`.timeline-comment-header-text:not(.has-full-name) a[href="/${user}"]`);
```

Comment:
This gets the list of names and then the `.each` updates the DOM, so I think the one change is sufficient.